testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/CamundaProcessTestContext.java

```diff
@@ -108,21 +108,77 @@ public interface CamundaProcessTestContext {
    */
   JobWorkerMock mockJobWorker(final String jobType);
 
+  /**
+   * Mocks a child process with the specified ID and sets the provided variables.
+   *
+   * @param childProcessId the ID of the child process to mock
+   * @param variables a map of variables to set for the mocked child process
+   */
+  void mockChildProcess(final String childProcessId, Map<String, Object> variables);
```

Comment:
:wrench: We should provide an overloaded method without a `variables` parameter. In some cases, you don't need variables from the child process.